src/NuGetGallery/Services/ContentObjectService.cs

```diff
@@ -2,9 +2,18 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Linq;
```

Comment:
Are all these `using`s still necessary? If not, please trim out unnecessary ones.